packs/data/pathfinder-bestiary-3.db/giant-opossum.json

```diff
@@ -239,7 +294,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p><strong>Trigger</strong> The opossum is reduced below 15 HP</p>\n<hr />\n<p><strong>Effect</strong> The opossum collapses. It is @UUID[Compendium.pf2e.conditionitems.Flat-Footed]{Flat-Footed} and can use actions that require only its mind, but any other action ends the ruse.</p>\n<p>A successful @Check[type:perception|dc:18] check to Seek or Medicine check to Recall Knowledge is required to determine that the animal is not, in fact, dead.</p>"
+                    "value": "<p><strong>Trigger</strong> The opossum is reduced below 15 HP</p>\n<hr />\n<p><strong>Effect</strong> The opossum collapses. It is @UUID[Compendium.pf2e.conditionitems.Flat-Footed]{Flat-Footed} and can use actions that require only its mind, but any other action ends the ruse.</p>\n<p>A successful @Check[type:perception|dc:18|action:seek] check to @UUID[Compendium.pf2e.actionspf2e.Seek]{Seek} or @Check[type:medicine|dc:18|action:recall-knowledge] check to @UUID[Compendium.pf2e.actionspf2e.Recall Knowledge (Medicine)]{Recall Knowledge (Medicine)} is required to determine that the animal is not, in fact, dead.</p>"
```

Comment:
The recall knowledge stuff just got consolidated so this may need fixed.

Reply:
Rebased